.gitignore

```diff
@@ -0,0 +1 @@
+vendor
```

Comment:
Couple of suggestions:

* Github hints a file must end in a blank line (hence the **red** symbol). Discussion [here](http://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)
* I would also added a `/` to the end of `vendor`, to become `vendor/`
* Add `composer.phar` to the ignore list